### DISCUSSION TITLE
change gas when dtr hits 12%

### DISCUSSION THREAD

**Discussion by ErezMinuto** [artisan-roaster-scope/artisan#1327] [Q&A]
Is there a way of triggering a GAS when the current DTR in the roast matches the DTR let say 12%?

**Comment by MAKOMO** [comment]
This is in detailed explained in the post [Symbolic Formulas: Basics, New Variables and Applications](https://artisan-roasterscope.blogspot.com/2019/11/symbolic-formulas-basics-new-variables.html) in Section _3.4 Symbolic Formulas and Alarms_, paragraph _SET BURNER TO 20% ONCE DevRatio REACHES 12%_.
